NEWS.md

```diff
@@ -1,5 +1,18 @@
 # testthat (development version)
 
+* Snapshots now use rlang to print error and warning messages,
+  including the `Error:` and `Warning:` prefixes. This means the
+  `call` field of conditions is now displayed in snapshots if present.
+
+* By default, snapshots no longer show the class of errors, warnings,
+  and messages. Set the new argument `cnd_class = TRUE` to restore the
+  old behaviour.
+
+  We decided to change the default to prevent distracting snapshot
+  diffing as upstream packages add error classes. For instance, the
+  development version of R is adding classes to basic errors.
```

Comment:
I like mentioning this as a reason for this change. I just had a case where I was using `warn_deprecated()` from `compat-lifecycle.R`, which calls `.Deprecated()`. My GHA 3.5.3 build failed a snapshot test because the error class used to be `"simpleWarning"` but they have recently added the class `"deprecatedWarning"`